src/components/AddQuote.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import './AddQuote.css'
+
+class AddQuote extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			autor: '',
+			quote: ''
+		}
+		this.addQuote = this.addQuote.bind(this);
+		this.handleInput = this.handleInput.bind(this);
+		this.deleteQuote = this.deleteQuote.bind(this);
+	}
+
+	handleInput(event) {
+		const target = event.target;
+		const value = target.value;
+		const name = target.name;
+		this.setState({
+			[name]: value
+		});
+	}
+
+	addQuote(e) {
+		e.preventDefault();
+		fetch("https://baza-podataka.herokuapp.com/dodaj-citat/", {
+			method: 'post',
+			headers: {
+				'Accept': 'application/json, text/plain, */*',
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify({
+				autor: this.state.autor,
+				sr: this.state.quote
+			})
+		})
+  alert(`Quote by: "${this.state.autor}" has been updated to quote base.
```

Comment:
ne koristi alert funkciju u produkciji, to se smatra ofanzivnim.

Reply:
plus, poruka je lažna, još uvek se ne zna da li je poruka sačuvana u bazi. treba sačekati odgovor od fetch funkcije.